src/zkevm_specs/evm/execution/create.py

```diff
@@ -0,0 +1,149 @@
+from zkevm_specs.evm.opcode import Opcode
+from zkevm_specs.util.arithmetic import FQ, RLC
+from zkevm_specs.util.hash import EMPTY_CODE_HASH, EMPTY_HASH
+from zkevm_specs.util.param import (
+    GAS_COST_CREATE,
+    N_BYTES_GAS,
+    N_BYTES_U64,
+)
+from ...util import (
+    CALL_CREATE_DEPTH,
+)
+from ..instruction import Instruction, Transition
+from ..table import RW, CallContextFieldTag, AccountFieldTag, CopyDataTypeTag
+
+
+def create(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+    is_create, is_create2 = instruction.pair_select(opcode, Opcode.CREATE, Opcode.CREATE2)
+    instruction.responsible_opcode_lookup(opcode)
+    callee_call_id = instruction.curr.rw_counter
+
+    value = instruction.stack_pop()
+    offset = instruction.stack_pop()
+    size = instruction.stack_pop()
+    contract_address = instruction.stack_push()
+
+    depth = instruction.call_context_lookup(CallContextFieldTag.Depth)
+    tx_id = instruction.call_context_lookup(CallContextFieldTag.TxId)
+    caller_address = instruction.call_context_lookup(CallContextFieldTag.CallerAddress)
+    nonce, nonce_prev = instruction.account_write(caller_address, AccountFieldTag.Nonce)
+    is_success = instruction.call_context_lookup(CallContextFieldTag.IsSuccess)
+    is_static = instruction.call_context_lookup(CallContextFieldTag.IsStatic)
+    reversion_info = instruction.reversion_info()
+    contract_address = instruction.generate_contract_address(caller_address, nonce_prev)
```

Comment:
We might need to rename this variable to compare it with the stack push one (stack push one should be `is_success * instruction.generate_contract_address(...)`).

Reply:
fixed in [80d39e8](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/355/commits/80d39e83047d4084c7c9b29ae51f9c9c0b1a9d42)